src/condition.cpp

```diff
@@ -952,6 +1092,41 @@ bool ConditionDamage::setParam(ConditionParam_t param, int32_t value)
 	return ret;
 }
 
+int32_t ConditionDamage::getParam(ConditionParam_t param)
+{
+	switch (param) {
+		case CONDITION_PARAM_OWNER:
+			return owner;
+
+		case CONDITION_PARAM_FORCEUPDATE:
+			return forceUpdate ? 1 : 0;
+
+		case CONDITION_PARAM_DELAYED:
+			return delayed ? 1 : 0;
+
+		case CONDITION_PARAM_MAXVALUE:
+			return maxDamage;
+
+		case CONDITION_PARAM_MINVALUE:
+			return minDamage;
+
+		case CONDITION_PARAM_STARTVALUE:
+			return startDamage;
+
+		case CONDITION_PARAM_TICKINTERVAL:
+			return tickInterval;
+
+		case CONDITION_PARAM_PERIODICDAMAGE:
+			return periodDamage;
+
+		case CONDITION_PARAM_FIELD:
+			return field ? 1 : 0;
+
+		default:
+			return Condition::getParam(param);;
```

Comment:
On an otherwise spotless pull request
```suggestion
			return Condition::getParam(param);
```